src/ralph/utils.py

```diff
@@ -84,3 +85,14 @@ def now():
     """Return the current UTC time in ISO format"""
 
     return datetime.datetime.now(tz=datetime.timezone.utc).isoformat()
+
+
+def nested_get(dict_: dict, keys: list[str]):
```

Comment:
What about changing the function name and also it's signature?

```suggestion
def get_dict_value_from_path(dict_: dict, path: str, sep: str = ">"):
    keys = path.split(sep)
    # [...]
```

Reply:
Good for the name change) For the signature change I'm hesitating, this function will be called many times per event and having to split the path at every time can be avoided by splitting the path once at the beginning.
WDYT ?